mongodb/README.md

```diff
@@ -0,0 +1,46 @@
+# About
+
+This image manages mongodb databases backups in an s3 object store.
+
+# Usage
+
+The image has different scripts used for separate purposes (they are intended to run in tandem as separate deamons):
+
+> NOTE: To be connected to the right environment, simply set the **MONGO_URL** environment variable.
+
+- /opt/backup.py: Create a backup copy of a mongodb database in s3. It takes the follow environment variables as input for s3:
+    - BACKUP_CMD: Backup command to execute, excluding the backup path (should be something like `mongodump --uri=$MONGO_URL --archive >`).
```

Comment:
You could define it as a template like so:
```
`mongodump --uri={mongodb_uri}`
```
Then, you could have a default and if end users wish to override it, they would just have to have to include the `{mongodb_uri}` notation in their override.

Reply:
Not sure I understand what you had in mind... where would you define the template?

Reply:
See my comment below.